README.md

```diff
@@ -28,6 +28,25 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 ```
 
+### specify a file
```

Comment:
```suggestion
### Specify a file
```

Here IMHO, we should also specify that in case this varibale is filled up, we override the message. 

For instance : 

```
      - name: Comment PR
        uses: thollander/actions-comment-pull-request@v1
        with:
          message: Hello !
          filePath: /path/to/file.txt
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
``` 
Would go for the file and not the message. 
Should it really be the default behaviour btw ? I think `message` should have more power than `filePath` in case it's present. Then I would go for an `if (filePath && !message)`